src/WebUI/Components/KubeSummary.tsx

```diff
@@ -175,6 +185,7 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
                     key={format("dc-{0}", this.state.namespace || "")}
                     onItemInvoked={this._onDeploymentItemInvoked}
                 />
+                {this.state.showOrphanPods && this.getOrphanPods()}

```

Comment:
should we move this deploymentscomponent, or rename deploymentscomponent to workloadscomponent?